Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -59,6 +59,7 @@
 from L1Trigger.Configuration.L1Trigger_EventContent_cff import *
 from RecoVertex.BeamSpotProducer.BeamSpot_EventContent_cff import *
 from CommonTools.ParticleFlow.EITopPAG_EventContent_cff import EITopPAGEventContent
+from RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff import *
```

Comment:
the comment above was to the rest of the file
The RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff does not belog in EventContent_cff